tests/integrations/python/tests/test_openai.py

```
@@ -3042,3 +3042,116 @@ def test_53_web_search_streaming(self, provider, model, vk_enabled):
         
         print(f"✓ Web search (streaming) test passed!")
 
+    @pytest.mark.parametrize(
+        "provider,model,vk_enabled", get_cross_provider_params_with_vk_for_scenario("code_execution")
+    )
+    def test_64_code_execution_math(self, test_config, provider, model, vk_enabled):
```

Ruff (0.14.11)

Unused method argument: `test_config`

(ARG002)